includes/Checker/Check_Categories.php

```diff
@@ -46,13 +40,35 @@ public static function get_categories() {
 	 * @return array An array of category labels.
 	 */
 	public static function get_category_labels() {
-		return array(
+		return self::get_check_categories();
+	}
+
+	/**
+	 * Returns an array of check categories.
+	 *
+	 * @since 1.0.2
+	 *
+	 * @return array An array of check categories.
+	 */
+	public static function get_check_categories() {
```

Comment:
Confusing to have get_check_categories and get_categories methods. Can we rename them?

Reply:
* `get_categories`
* `get_category_slugs`
* `get_category_labels`